docs/community.rst

```diff
@@ -1,18 +1,22 @@
 Community Resources
 ===================
 
-Below are a few community resources about Rally. If you find an interesting article, talk or custom tracks, `raise an issue <https://github.com/elastic/rally/issues>`_  or `open a pull request <https://github.com/elastic/rally/pulls>`_.
+Below are a few community resources about Rally. If you find an interesting article, talk, or custom tracks, `raise an issue <https://github.com/elastic/rally/issues>`_  or `open a pull request <https://github.com/elastic/rally/pulls>`_.
 
-Talks
------
+Videos
+------
 
 .. raw:: html
 
    <iframe width="560" height="315" src="https://www.youtube.com/embed/HriBY2zoChw" frameborder="0" allowfullscreen></iframe>
 
 .. raw:: html
 
-   <iframe width="560" height="315" src="https://www.youtube.com/embed/Fm7fxE4kxPI" frameborder="0" allowfullscreen></iframe>
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/-jgmnpG_P-Y" frameborder="0" allowfullscreen></iframe>
```

Comment:
I played around with headers to have titles for the videos, but the Youtube embed overlay does a pretty good job